services/github/README.md

```diff
@@ -89,13 +89,13 @@ docker-compose -f docker-compose-prod.yml run github-service python manage.py te
 
 # Reference
 
-### Kill All Running Docker Instances
+### Kill all the Running Docker Instances
```

Comment:
All should still be capitalized.

Reply:
Thank you for reviewing! Resolved the issue.